src/hooks/binance/updateUpbitListWithBinance.ts

```diff
@@ -0,0 +1,23 @@
+import { IUpbitTicker } from 'hooks/upbit/useUpbitTicker';
+import { IBinanceTicker } from './useBinanceTicker';
+export function updateUpbitListWithBinance(
+  list: IUpbitTicker[],
+  binanceTickers: IBinanceTicker[],
+): IUpbitTicker[] {
+  const removeUSDT = (symbol: string) => {
+    return symbol.replace('USDT', '');
+  };
+
+  return list.map((item) => {
+    const binanceTicker = binanceTickers?.find(
+      (ticker) => removeUSDT(ticker.s) === item.code?.split('-')[1],
+    );
+    if (binanceTicker) {
+      return {
+        ...item,
+        binancePrice: binanceTicker.c,
+      };
+    }
```

Comment:
여기에 일치하지 않아서 바이낸스 티커가 없는 경우도 추가해줘야하지 않을까요?
```
else {
 바이낸스 티커가 없을 때 로직
}
```

Reply:
> Reference in ne

```suggestion
    if (binanceTicker) {
      return {
        ...item,
        binancePrice: binanceTicker.c,
      };
    }
    return item;
```

여기서 return item을 해주고있기 때문에 이게 곧 else 와 다름이 없습니다!
일치하지 않으면 그냥 item만 반환합니다!

Reply:
음 els문이 없으면 바이낸스 티커가 없을 때는 업비트 데이터에 binancePrice 값이 없는 것 아닌가용?

Reply:
> 음 els문이 없으면 바이낸스 티커가 없을 때는 업비트 데이터에 binancePrice 값이 없는 것 아닌가용?

네 맞습니다